source/sass/type.scss

```diff
@@ -50,29 +50,6 @@ a {
   line-height: 1.5;
 }
 
-@mixin secondary-nav-link($active-class-name) {
```

Comment:
We don't need this mixin anymore as now it's only `.multipage-link` that's using it. So I combined the rules into `.multipage-link`